magicblock-config/src/ledger.rs

```diff
@@ -123,20 +119,24 @@ impl From<LedgerResumeStrategy> for LedgerResumeStrategyConfig {
 pub struct LedgerResumeStrategyConfig {
     #[derive_env_var]
     #[serde(default)]
-    pub variant: LedgerResumeStrategyType,
+    pub kind: LedgerResumeStrategyType,
     #[derive_env_var]
     #[clap_from_serde_skip]
     #[serde(default)]
     pub reset_slot: Option<u64>,
```

Comment:
ditto not sure if you need this `Option` since it'd default to `0` if not provided

Reply:
Realized in the next part why you need option for verification. Kinda sad though :)